client/me/purchases/manage-purchase/style.scss

```diff
@@ -9,17 +9,25 @@
 
 	&.is-expired {
 		background: $gray-light;
-		margin-bottom: 0;
 
 		.manage-purchase__detail,
 		.manage-purchase__detail-label,
 		.manage-purchase__settings-link,
 		.manage-purchase__title,
-		.manage-purchase__subtitle {
+		.manage-purchase__subtitle,
+		.manage-purchase__remove-text {
```

Comment:
I think adding all these different classes will result in really large CSS. We can probably simplify this a bit. I'll open a PR.